presto-docs/src/main/sphinx/release/release-0.222.rst

```diff
@@ -0,0 +1,45 @@
+=============
+Release 0.222
+=============
+
+General Changes
+---------------
+* Fix incorrect results when dividing certain decimal numbers.
+* Fix query failure for certain join queries with "Equi criteria are empty,
+  so INNER join should not have PARTITIONED distribution type".
+* Add support for recoverable grouped execution. This can be enabled with the
+  ``recoverable_grouped_execution`` session property.
+* Add support for grouped execution for ScanFilterProject-only queries. This
+  can be enabled with the ``grouped_execution_for_eligible_table_scans`` session
+  property.
+* Add config property ``max-concurrent-materializations`` and session property
+  ``max_concurrent_materializations`` to limit the number of plan sections that
+  will run concurrently when using materialized exchanges.
+* Add support for computing :func:`approx_distinct` over BingTile values.
+* Add :func:`merge_hll` which performs a scalar merge of an array of HyperLogLogs.
+* Add bitwise shift operations, :func:`bitwise_arithmetic_shift_right`,
+  :func:`bitwise_logical_shift_right` and :func:`bitwise_shift_left`.
+* Add completed and total lifespans on Web UI.
+
+Hive Connector Changes
+----------------------
+* Fix failures for ``information_schema`` queries when a table has an invalid storage format.
+* Improve query execution time over bucketed table with large buckets.
```

Comment:
Does this mean large # of buckets or buckets whose files are large?

Reply:
buckets whose files are large.